### PR TITLE
chore: e2e test improvements

### PR DESCRIPTION
**Pull request by daviroo**

### CODE REVIEW

File: android/fastlane/Fastfile [hunk 1]
```diff
@@ -76,7 +76,7 @@ platform :android do
     upload_to_browserstack_app_automate(
       browserstack_username: ENV["BROWSERSTACK_USER"],
       browserstack_access_key: ENV["BROWSERSTACK_ACCESS_KEY"],
-      file_path: "./app/build/outputs/apk/release/app-universal-release.apk"
+      file_path: ENV["GRADLE_APK_OUTPUT_PATH"]
```
Comment (nicolasburtey) [inline android/fastlane/Fastfile:79]: I think I had set this before but `GRADLE_APK_OUTPUT_PATH` was not consistently returning the universal build, fyi. maybe I'm mistaking but something to keep in mind.

File: .circleci/config.yml [hunk 2]
```diff
@@ -252,3 +254,5 @@ workflows:
     jobs:
       - test_android
       - test_ios
+
+# VS Code Extension Version: 1.5.1
```
Comment (nicolasburtey) [inline .circleci/config.yml:258]: not intended?

File: e2e/06-payments-receive-flow.e2e.spec.ts [hunk 3]
```diff
@@ -81,7 +81,7 @@ describe("Receive Payment Flow", async () => {
       await shareButton.waitForDisplayed({ timeout: 8000 })
       await shareButton.click()
       const invoiceSharedScreen = await $('//*[contains(@name,"lntbs")]')
-      const invoiceSharedScreenElement = await invoiceSharedScreen.waitForDisplayed({
+      await invoiceSharedScreen.waitForDisplayed({
```
Comment (nicolasburtey) [inline e2e/06-payments-receive-flow.e2e.spec.ts:84]: there is another missing await there: https://github.com/GaloyMoney/galoy-mobile/pull/917/files#diff-6172ef87d6da592396503b953060c2c4cc11286af02fcb450fd398005b40e054R16

File: android/fastlane/Fastfile (hunk 1 above)
Comment (daviroo) [reply android/fastlane/Fastfile:79]: Testing this it seems to work

File: e2e/06-payments-receive-flow.e2e.spec.ts (hunk 3 above)
Comment (daviroo) [reply e2e/06-payments-receive-flow.e2e.spec.ts:84]: Fixed

File: .circleci/config.yml (hunk 2 above)
Comment (daviroo) [reply .circleci/config.yml]: Removed

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**